docs/content/users/usage/commands.md

```diff
@@ -195,7 +195,7 @@ ddev auth ssh
 
 Flags:
 
-* `--ssh-key-path`, `-d`: Full path to SSH key directory.
+* `--ssh-key-path`, `-d`: Full path to SSH key directory or file.
```

Comment:
If we add aliases for -f and --ssh-key-file then this would get updated too.

Reply:
I added a new flag.